doc/doc/basics/configuration.md

```diff
@@ -321,79 +319,7 @@ The following directives are only for the `json_extractor`, `tsv_extractor`, and
   guide](extractors.html) for details about the requirements for the UDF for
   different styles of extractors.
 
-In the following subsections we present the directives that are specific to each
-extractor style, if any.
-
-#### <a name="json" href="#"></a> json\_extractor / tsv\_extractor
-
-- `parallelism`: specifies the number of instances of this extractor to be
-  executed in parallel. Tuples generated by the `input` query are sent (in
-  batches of size controlled by the `input_batch_size` directive) to each
-  instance in a round-robin fashion. This is an optional directive. Example
-  usage:
-
-    ```bash
-    myExtractor {
-      # ...
-      style: "json_extractor"
-      # ...
-      parallelism: 5
-      # ...
-    }
-    ```
-
-- `input_batch_size`: specifies the size of the batch of tuples to feed to each
-  instance of the extractor at each round-robin iteration. This is an optional directive. **The default value is 10000.**  Example usage:
-
-    ```bash
-    myExtractor {
-      # ...
-      style: "json_extractor"
-      # ...
-      parallelism: 5
-      # ...
-      input_batch_size: 5000
-      # ...
-    }
-    ```
-
-- `output_batch_size`: specifies how many tuples in the output of the extractor
-  are inserted into the `output_relation` at once. If the tuples are large, a
-  smaller value may avoid incurring in out of memory errors. This is an optional directive. **The default value is 50000.** Example usage:
-
-    ```bash
-    myExtractor {
-      # ...
-      style: "json_extractor"
-      # ...
-      output_batch_size: 5000
-      # ...
-    }
-    ```
-
-#### sql_extractor
-
-- `sql`: specifies the SQL command to execute. This option is mandatory for this
-  extractor style. Example usage:
-
-    ```bash
-    myExtractor {
-      style: "sql_extractor"
-      sql: """INSERT INTO titles VALUES (1, 'Moby Dick')"""
-    }
-    ```
-
-#### cmd_extractor
-
-- `cmd`: specifies the shell command to execte. This option is mandatory for
-  this extractor style. Example usage:
-
-    ```bash
-    myExtractor {
-      style: "cmd_extractor"
-      cmd: """python words.py"""
-    }
-    ```
+For more information about different types of extractors can be found in ['Writing extractors' guide](extractors.html).
```

Comment:
This seems to be just repeating [L219](https://github.com/HazyResearch/deepdive/pull/396/files#diff-c2706fbfe92b4785b1ecee704a4952c8R219).  I think what needs to be mentioned/summarized here is that there are other directives (e.g., `parallelism`, `sql`, `cmd`) for different `style`.